docs/intro-to-lora.md

```diff
@@ -141,6 +141,31 @@ Now that we have set up the gateways and they can reach the internet, it's time
 
 <span class="tips">**Tip:** LORIOT has a 'Gateway Tap', which you can access from your gateway page. This allows you to see what messages are being picked up by the gateway - very useful when debugging.</span>
 
+### Automatically starting the LORIOT binary when the gateway starts
+
+#### Kerlink IoT station
+
+If you follow the installation steps in the LORIOT documentation, the binary will automatically start whenever the gateway boots. No additional steps required.
+
+#### Multitech Conduit
+
+1. Log in over SSH or via the serial port.
+1. Create a new file ``/etc/init.d/start-loriot.sh``. 
+1. Edit this file (f.e. via `nano /etc/init.d/start-loriot.sh`) and add the following content:
+
+    ``cd /home/root/ && ./loriot_multitech_conduit_mCard_USB_1.0.1``
+
+    * If you extracted the LORIOT binary somewhere else, edit the path. 
+    * It's important to `cd` into the folder first, otherwise LORIOT cannot find its certificate.
```

Comment:
I would say "we're CDing into the folder first" and not "it's important to"; the current woring might make people who didn't pay attention to the code snippet think they still need to do something.